addons/tagging/functions/fnc_tagTestingThread.sqf

```diff
@@ -27,7 +27,7 @@ GVAR(tagsToTest) = GVAR(tagsToTest) select {
 
     // If there's no intersections
     if (_intersections isEqualTo []) then {
-        TRACE_1("No intersections, deleting:",_tag);
```

Comment:
Is there an issue with whitespaces following commas in strings?

Reply:
No. Probably regex replacement.

Reply:
commas are ignored in strings in macros
```
#define TEST_MACRO(A) [A]
x2 = TEST_MACRO("TEST, STRING");
```
`["TEST STRING"]`

Reply:
Revert the last two commits then?

Reply:
no your fixes were great
`Won't frag,but will spall`
would be even more unreadable
`Won't fragbut will spall`